.buildkite/commands/lint-localized-strings-format.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eu
+
+echo "--- :rubygems: Setting up Gems"
+install_gems
+
+echo "--- :cocoapods: Setting up Pods"
+install_cocoapods
+
+echo "--- :writing_hand: Copy Files"
+mkdir -pv ~/.configure/wordpress-ios/secrets
+cp -v fastlane/env/project.env-example ~/.configure/wordpress-ios/secrets/project.env
+
+echo "--- Lint Localized Strings Format"
+bundle exec fastlane generate_strings_file_for_glotpress skip_commit:true
```

Comment:
I'm debating whether to put in the time to somehow capture the `genstrings` error, if any, and to annotate the build with it, vs banking on this to be a low occurrence kind of failure and accepting a less polished UX.

Reply:
I'm not sure how easy it would be to capture the error from `genstrings` here anyway 🤔 

Maybe one easy way would be to store the logs of `bundle exec fastlane` command (e.g. `| tee logs.txt`) then `grep` for `genstrings: error:` lines in it?
```
bundle exec fastlane generate_strings_file_for_glotpress skip_commit:true | tee genstrings-logs.txt
if [[ $? -ne 0 ]]; then
  # Previous step errored, extract the errors in an easy-to-find section
  echo "--- Report genstrings errors"
  echo "Found errors when trying to run genstrings to generate the `.strings` files from `*LocalizedStrings` calls."
  echo "See below for a summary (or previous section for more details)."
  grep 'genstrings: ' 'genstrings-logs.txt'
  grep 'genstrings: ' 'genstrings-logs.txt' | buiildkite-agent annotate --style error --context genstrings
fi
```

If that works, that could be an easy enough addition to the script without too much complication. If that doesn't work, then probably not worth spending too much time on it though.

Reply:
Thanks! I'll have a go a it.

I didn't go down the piping output option because I didn't want to lose the logs in the Buildkite UI. I had forgotten about the `tee` command 🤦‍♂️

Reply:
@AliSoftware addressed in 5e355c38f6.

What do you think of extracting this in the Buildkite plugin? I don't like how it depends on a lane name, which could vary across projects. But... we have a lot of convention over configuration already, and we can have proper error messages around that in the plugin command.

Reply:
Good idea to extract in plugin commands.

I agree that we have a lot of convention over configuration but I'm not sure the name of this lane is one that is currently consistent across repos at the moment 🤔 Maybe we can accept the lane name as optional parameter (`${1:-generate_strings_file_for_glotpress}`) to cover our backs (instead of failing if convention isn't followed in a repo)?